components/account/home/ApiKeys.vue

```diff
@@ -8,84 +8,55 @@ const { account } = storeToRefs(store);
 const apiKey = computed(() => account.value?.apiKey ?? '');
 const apiSecret = computed(() => account.value?.apiSecret ?? '');
 const hasApiKeys = computed(() => apiKey.value && apiSecret.value);
-const showKey = ref(false);
-const showSecret = ref(false);
 
 const regenerateKeys = async () => await store.updateKeys();
 
-const css = useCssModule();
+const { t } = useI18n();
 </script>
 
 <template>
-  <CardContainer>
-    <TextHeading :class="css.heading" subheading>API Credentials</TextHeading>
-    <div :class="css.entry">
-      <InputField
-        id="api-key"
-        :class="css.input"
-        :model-value="apiKey"
-        :type="showKey ? 'text' : 'password'"
-        label="API Key"
-        readonly
-      />
-      <VisibilityButton v-model="showKey" :class="css.toggle" />
-      <div :class="css.col">
-        <CopyButton :class="css.button" :model-value="apiKey" />
+  <div>
+    <div class="mb-6 flex justify-between items-center">
+      <div class="text-h6">{{ t('account.api_keys.title') }}</div>
+      <div>
+        <RuiButton color="primary" @click="regenerateKeys()">
+          <template #prepend>
+            <RuiIcon name="refresh-line" />
+          </template>
+          {{ hasApiKeys ? t('actions.regenerate') : t('actions.generate') }}
+        </RuiButton>
       </div>
     </div>
-    <div :class="css.entry">
-      <InputField
-        id="api-secret"
-        :class="css.input"
+    <div class="grid md:grid-cols-2 gap-4">
+      <RuiRevealableTextField
+        id="username"
```

Comment:
we have to change the id here and below :)

Reply:
on it.

Reply:
done